detox/android/detox/src/main/java/com/wix/detox/DetoxActionHandlers.kt

```diff
@@ -45,12 +45,11 @@ class InvokeActionHandler(
         private val wsClient: WebSocketClient)
     : DetoxActionHandler {
 
-    private val validResultData = mapOf<String, Any>(Pair("result", "(null)"))
 
     override fun handle(params: String, messageId: Long) {
         try {
-            methodInvocation.invoke(params)
-            wsClient.sendAction("invokeResult", validResultData, messageId)
+            val invocationResult = methodInvocation.invoke(params)
+            wsClient.sendAction("invokeResult", mapOf<String, Any?>(Pair("result", invocationResult)), messageId)
```

Comment:
BTW need to check what the iOS equiv. of this does. @LeoNatan do you have an idea?

Reply:
Need more context here pls.

Reply:
@LeoNatan, @omerwin's code here simply returns the invocation's result, which hasn't been done until now, it seems. The complementary code was implemented on the JS side as well - I think we should take a moment to see what happens on iOS in that sense...

Reply:
all the test seems to be fine, in ios until now we didn't get any response so I suppose it will be fine